io/plugins/eu.esdihumboldt.hale.io.appschema.test/src/eu/esdihumboldt/hale/io/geoserver/rest/NamespaceResourceTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.geoserver.rest;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+
+import eu.esdihumboldt.hale.io.geoserver.Namespace;
+import eu.esdihumboldt.hale.io.geoserver.ResourceBuilder;
+
+/**
+ * Tests the correct serialization of a namespace resource.
+ * 
+ * @author Stefano Costa, GeoSolutions
+ */
+public class NamespaceResourceTest extends ResourceTestBase {
+
+	private static final String TEST_ID = "namespace-1234";
+	private static final String TEST_PREFIX = "test_ns";
+	private static final String TEST_URI = "http://www.example.com";
```

Comment:
This is unused an generates a compiler warning. If it's still required, could you add a `@SuppressWarnings("unused")` to prevent the compiler warning?

Reply:
Is the TEST_URI of ``WorkspaceResourceTest`` class that is actually not used, fixed.